src/main/java/com/regnosys/rosetta/common/serialisation/projectiondata/ProjectionDataSet.java

```diff
@@ -45,8 +43,14 @@ public String toString() {
                 .add("dataSetName='" + getDataSetName() + "'")
                 .add("inputType='" + getInputType() + "'")
                 .add("expectedType='" + getExpectedType() + "'")
-                .add("applicableReports=" + applicableProjection)
+                .add("applicableProjection=" + applicableProjection)
+                .add("applicableProjections=" + applicableProjections)
                 .add("data=" + getData())
                 .toString();
     }
+
+    public SortedSet<String> getApplicableProjections() {
+        return this.applicableProjections;
```

Comment:
This should just return a List the underlying member field can remain a sorted set still. It should also add the singular applicableProjection to the sorted set before returning as a list.

Reply:
Why is this still returning the set? We only need one getter for applicableProjections